{"uid":"efc5ea3a92d9068d","name":"Checks that remote component card appears after loading","fullName":"Checks that remote component card appears after loading","historyId":"ac4068d6682dfbd9b4ca77884385be9f","time":{"start":1680283065913,"stop":1680283067443,"duration":1530},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","steps":[{"name":"\"before each\" hook","time":{"start":1680283065913,"stop":1680283066191,"duration":278},"status":"passed","steps":[{"name":"visit http://localhost:4173","time":{"start":1680283066049,"stop":1680283066190,"duration":141},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"uncaught exception TypeError: ReactDOM.createRoot is not a function","time":{"start":1680283066193,"stop":1680283067442,"duration":1249},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"get [data-e2e=\"APP__CARD\"]","time":{"start":1680283067440,"stop":1680283067440,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"assert expected **undefined** to have a length of **1** but got **0**","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067441,"stop":1680283067441,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067442,"stop":1680283067442,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1680283067442,"stop":1680283067442,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[{"uid":"6f0b75d10bcc02a2","name":"e2e/tests/runAll.cy.ts:2023-03-31T17:17:45.545Z","source":"6f0b75d10bcc02a2.png","type":"image/png","size":167924},{"uid":"d3efedd2a99996ed","name":"e2e/tests/runAll.cy.ts:2023-03-31T17:17:47.082Z:attempt-1","source":"d3efedd2a99996ed.png","type":"image/png","size":155762},{"uid":"8e251f6a098e7e39","name":"video recording","source":"8e251f6a098e7e39.mp4","type":"video/mp4","size":4712007}],"parameters":[],"stepsCount":20,"attachmentsCount":3,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"Vite React Microfrontends"},{"name":"suite","value":"It checks host app"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":17,"broken":0,"skipped":0,"passed":97,"unknown":0,"total":114},"items":[{"uid":"34d129731a167476","reportUrl":"https://module-federation.github.io/module-federation-examples/584//#testresult/34d129731a167476","status":"passed","time":{"start":1680280122941,"stop":1680280126047,"duration":3106}},{"uid":"22dc7160043eaa9a","reportUrl":"https://module-federation.github.io/module-federation-examples/581//#testresult/22dc7160043eaa9a","status":"passed","time":{"start":1680191664599,"stop":1680191667639,"duration":3040}},{"uid":"814ad1594d2c2675","reportUrl":"https://module-federation.github.io/module-federation-examples/577//#testresult/814ad1594d2c2675","status":"passed","time":{"start":1680257679111,"stop":1680257682461,"duration":3350}},{"uid":"3c6169489ef121c3","reportUrl":"https://module-federation.github.io/module-federation-examples/576//#testresult/3c6169489ef121c3","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680256828424,"stop":1680256829702,"duration":1278}},{"uid":"c6eba2a75a98cc2a","reportUrl":"https://module-federation.github.io/module-federation-examples/566//#testresult/c6eba2a75a98cc2a","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680220351511,"stop":1680220353401,"duration":1890}},{"uid":"dc398330859584d5","reportUrl":"https://module-federation.github.io/module-federation-examples/563//#testresult/dc398330859584d5","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680193316357,"stop":1680193317593,"duration":1236}},{"uid":"a16619037cd3da16","reportUrl":"https://module-federation.github.io/module-federation-examples/562//#testresult/a16619037cd3da16","status":"passed","time":{"start":1680191664599,"stop":1680191667639,"duration":3040}},{"uid":"571c551b32e4ef04","reportUrl":"https://module-federation.github.io/module-federation-examples/554//#testresult/571c551b32e4ef04","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680186006384,"stop":1680186007648,"duration":1264}},{"uid":"9ec8c2a6721ab50b","reportUrl":"https://module-federation.github.io/module-federation-examples/542//#testresult/9ec8c2a6721ab50b","status":"passed","time":{"start":1680175121390,"stop":1680175125346,"duration":3956}},{"uid":"e4c4931f7e5452b7","reportUrl":"https://module-federation.github.io/module-federation-examples/541//#testresult/e4c4931f7e5452b7","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680125924503,"stop":1680125926294,"duration":1791}},{"uid":"60f6a63687f1c8ef","reportUrl":"https://module-federation.github.io/module-federation-examples/539//#testresult/60f6a63687f1c8ef","status":"passed","time":{"start":1680066839089,"stop":1680066842924,"duration":3835}},{"uid":"e0fb32b9007a6788","reportUrl":"https://module-federation.github.io/module-federation-examples/532//#testresult/e0fb32b9007a6788","status":"passed","time":{"start":1679997262032,"stop":1679997265539,"duration":3507}},{"uid":"affcffde3d15bd97","reportUrl":"https://module-federation.github.io/module-federation-examples/530//#testresult/affcffde3d15bd97","status":"passed","time":{"start":1679971132182,"stop":1679971135679,"duration":3497}},{"uid":"976e816a7ce150ff","reportUrl":"https://module-federation.github.io/module-federation-examples/517//#testresult/976e816a7ce150ff","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1679957428375,"stop":1679957429430,"duration":1055}},{"uid":"2ecd7f733022505f","reportUrl":"https://module-federation.github.io/module-federation-examples/515//#testresult/2ecd7f733022505f","status":"passed","time":{"start":1679952215539,"stop":1679952219041,"duration":3502}},{"uid":"4b8b62f80e4d7f4b","reportUrl":"https://module-federation.github.io/module-federation-examples/512//#testresult/4b8b62f80e4d7f4b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1679913722520,"stop":1679913723721,"duration":1201}},{"uid":"1b51c1e3df84655c","reportUrl":"https://module-federation.github.io/module-federation-examples/511//#testresult/1b51c1e3df84655c","status":"passed","time":{"start":1679677159358,"stop":1679677162376,"duration":3018}},{"uid":"8050ea846af3876c","reportUrl":"https://module-federation.github.io/module-federation-examples/510//#testresult/8050ea846af3876c","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1679697184383,"stop":1679697185295,"duration":912}},{"uid":"1ba94cab4b93852c","reportUrl":"https://module-federation.github.io/module-federation-examples/508//#testresult/1ba94cab4b93852c","status":"passed","time":{"start":1679677159358,"stop":1679677162376,"duration":3018}},{"uid":"4d90459fd6e5d9bc","reportUrl":"https://module-federation.github.io/module-federation-examples/507//#testresult/4d90459fd6e5d9bc","status":"passed","time":{"start":1679675047025,"stop":1679675051330,"duration":4305}}]},"tags":[]},"source":"efc5ea3a92d9068d.json","parameterValues":[]}